services/user/PsiSpaceSys/ui/package.json

```diff
@@ -12,7 +12,8 @@
     },
```

Comment:
These changes are irrelevant currently, aren't they?

Reply:
that's right. reverted.